modules/library/coverage/src/main/java/org/geotools/image/ImageWorker.java

```diff
@@ -495,7 +559,29 @@ public final RenderedOp getRenderedOperation() {
         if (image instanceof RenderedOp) {
             return (RenderedOp) image;
         }
-        return NullDescriptor.create(image, getRenderingHints());
+        // Creating a parameter block
+        ParameterBlock pb = new ParameterBlock();
+        pb.setSource(image, 0);
+        // Executing the operation
+        return JAI.create("Null", pb, getRenderingHints());
+    }
+
+    /**
+     * Returns the {@linkplain #getRenderedImage rendered image} after null operation.
+     * 
+     * @return The rendered operation.
+     * 
+     * @see #getRenderedImage
+     * @see #getPlanarImage
+     * @see #getImageAsROI
+     */
+    public ImageWorker nullOp() {
```

Comment:
Explain why we do even have this method (by its name it might seem it does nothing, or sets all values in the coverage to null or... who knows)